src/utils/index.js

```diff
@@ -21,6 +21,9 @@ export const DEFAULT_CONFIG = {
     sidebar: {
       display: true,
     },
+    header: {
+      display: true,
+    },
```

Comment:
Whether or not to display the publish point should be part of config too? Under `tools`

Reply:
I'll make another PR after the Measure tool one that adds config options for tools (hiding/adding/deleting). This will be a part of that.